packages/next/src/views/List/handleServerFunction.tsx

```diff
@@ -1,45 +1,12 @@
-import type { I18nClient } from '@payloadcms/translations'
 import type { ListPreferences } from '@payloadcms/ui'
-import type {
-  ClientConfig,
-  ImportMap,
-  ListQuery,
-  PayloadRequest,
-  SanitizedConfig,
-  VisibleEntities,
-} from 'payload'
+import type { ListQuery, PayloadRequest, VisibleEntities } from 'payload'
 
+import { getClientConfig } from '@payloadcms/ui/utilities/getClientConfig'
```

Comment:
This import is wrong. Was it exported from the top level properly from `@payloadcms/ui`?

Reply:
Nope, because getClientConfig can only run on the server, we can't export it from the top-level `@payloadcms/ui.` Thus, I have exported it directly